lagunita/lms/static/sass/discussion/lms-discussion-main.scss

```diff
@@ -1,3 +1,10 @@
 @import 'lms/static/sass/discussion/lms-discussion-main';
 
 @import '../overrides';
+
+.discussion-user-profile-board {
+    .user-roles {
+	display:none !important;
```

Comment:
a few CSS/SASS style nits:
1. line 7 should have 8 spaces and line 6 should (and does) have 4 spaces.
2. space after colon
3. only use `!important` if absolutely necessary. Since our override gets applied after all the platform style, it shouldn't be necessary. (in CSS the last style definition wins.)